docs/koji.md

```diff
@@ -86,6 +88,17 @@ Data which is placed here includes:
 
 - `build.extra.typeinfo.operator-manifests.archive` (string): name of the archive containing operator manifest files
 
+# Type-specific source container build metadata
+
+The same note about `image` and `build.extra.typeinfo.image` from previous section applies here.
+
+Data which is placed here includes:
+
+- `build.extra.image.media_types`: the same as for image builds (see previous section)
+- `build.extra.image.sources_for_nvr` (str): koji N-V-R for the original image build
+- `build.extra.image.sources_signing_intent` (str): signing intent used to fetch sources
```

Comment:
Let's explicitly say "to fetch SRPMs" here. It might not be involved when it comes to fetching other types of source code.

Reply:
fixed